grammars/less.cson

```diff
@@ -277,7 +277,7 @@
     'name': 'keyword.operator.less'
   }
   {
-    'match': '\\s(\\=\\=|\\!\\=|\\<\\=|\\>\\=|\\<|\\>|not|or|and|when)\\s'
+    'match': '\\s(\\=\\=|\\!\\=|\\<\\=|\\>\\=|\\<|\\>|not|or|and|(when not)|when)\\s'
```

Comment:
I'm assuming that `when_not` and `when______not` are the same (assume that the underscores are spaces).  If so, then the regex should probably be rewritten as `when(\\s+not)?`.

Also, nothing needs to be escaped in this regex if you want to clean it up a bit.
Eg `\\s(==|!=|<=|>=|<|>|...`